specification/kubernetesconfiguration/resource-manager/readme.md

```diff
@@ -28,33 +28,32 @@ These are the global settings for the KubernetesConfiguration.
 title: SourceControlConfigurationClient
 description: KubernetesConfiguration Client
 openapi-type: arm
-tag: package-preview-2021-05
+tag: package-preivew-2021-05-01
 ```
 
 
-### Tag: package-preview-2021-05
+### Tag: package-preivew-2021-05-01
 
-These settings apply only when `--tag=package-preview-2021-05` is specified on the command line.
+These settings apply only when `--tag=package-preivew-2021-05-01` is specified on the command line.
 
-```yaml $(tag) == 'package-preview-2021-05'
+```yaml $(tag) == 'package-preivew-2021-05-01'
 input-file:
   - Microsoft.KubernetesConfiguration/preview/2021-05-01-preview/extensions.json
-  - Microsoft.KubernetesConfiguration/preview/2021-05-01-preview/kubernetesconfiguration.json
```

Comment:
Can you make sure to add back `Microsoft.KubernetesConfiguration/stable/2021-03-01/kubernetesconfiguration.json` here?

Reply:
Adding this causes LintDiff tool to break.  I've reached out to Swagger team for help; and will resolve based on discussion with them.